arangodb-net-standard/CollectionApi/CollectionApiClient.cs

```diff
@@ -71,11 +71,11 @@ public async Task<TruncateCollectionResult> TruncateCollectionAsync(string colle
         /// Gets count of documents in a collection
         /// GET/_api/collection/{collection-name}/properties
         /// </summary>
-        /// <param name="options"></param>
+        /// <param name="collectionName"></param>
         /// <returns></returns>
-        public async Task<GetCollectionCountResponse> GetCollectionCountAsync(GetCollectionCountOptions options)
+        public async Task<GetCollectionCountResponse> GetCollectionCountAsync(string collectionName)
         {
-            using (var response = await _transport.GetAsync(_collectionApiPath + "/" + options.CollectionName + "/count"))
+            using (var response = await _transport.GetAsync(_collectionApiPath + "/" + collectionName + "/count"))
```

Comment:
ArangoDB doesn't accept any collection name with non alphanumeric characters + '-' and '_' but if these are provided here it may mess up with the route. Should we consider this the user's responsibility? Do you think we need to encode?

Reply:
Could point. Perhaps we should have a validity check on all user queries and raise an Exception?

Reply:
Yes good point. Let's just URL encode anything we include in the URL path, that way at least the URL itself won't be misinterpreted.  I prefer that idea to validation of the collection name, which is responsibility of ArangoDB I think.  We just need to pass back whatever error response we get. 

Incidentally I think based on this we also should URL encode query parameters and path parameters passed in any other API methods.  What do you guys think?

Reply:
When it's a string I think it makes sense to encode it. But in most of the query parameters we use "1" or "0" anyway.